tests/foreman/api/test_syncplan.py

```diff
@@ -74,8 +76,8 @@ def valid_sync_interval():
     return ['hourly', 'daily', 'weekly', 'custom cron']
 
 
-class SyncPlanTestCase(APITestCase):
-    """Miscellaneous tests for sync plans."""
+class TestSyncPlan:
```

Comment:
Can we put all the test cases into a single Syncplan class?

Reply:
Hello @devendra104 , I have removed all the classes